models/traub_receptors.nestml

```diff
@@ -0,0 +1,202 @@
+"""
+Name: traub_psc_alpha - Traub model according to Borgers 2017.
```

Comment:
What do you think of renaming to "traub_cond_multisyn",  to fit the existing naming scheme better?

Reply:
I agree with renaming, why not. I work on the unit tests. 
Thank you.